thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/CleanupAndRegenerateAnomaliesTool.java

```diff
@@ -168,6 +204,10 @@ private void deleteRawResults(List<RawAnomalyResultDTO> rawResults) {
   private void deleteMergedResults(List<MergedAnomalyResultDTO> mergedResults) {
     LOG.info("Deleting merged results");
     for (MergedAnomalyResultDTO mergedResult : mergedResults) {
+      // Delete raw anomalies of the merged anomaly
```

Comment:
Could you elaborate that use case? 

It a raw anomaly exists in a certain time range, then detection function cannot generate any raw anomalies in that time range. So, it is a good practice to remove raw anomalies whenever its corresponding merged anomaly is removed.